### PR TITLE
onionsearch: add page

### PR DESCRIPTION
**Pull request by fazlearefin**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/common/onionsearch.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# onionsearch
+
+> Scrapes urls on different .onion search engines.
```
Comment (kbdharun) [inline pages/common/onionsearch.md:3]:
```suggestion
> Scrape URLs on different `.onion` search engines.
```

File: pages/common/onionsearch.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+> `onionsearch` requires tor proxy running on `localhost:9050`.
+> A tor enabled browser is needed to visit the .onion websites.
```
Comment (kbdharun) [inline pages/common/onionsearch.md:5]:
```suggestion
> Note: `onionsearch` requires tor proxy running on `localhost:9050` and a tor enabled browser is needed to visit the `.onion` websites.
```

File: pages/common/onionsearch.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+> More information: <https://github.com/megadose/OnionSearch>.
+
+- Request all the search engines for the word "Epstein":
```
Comment (kbdharun) [inline pages/common/onionsearch.md:8]:
```suggestion
- Request results from all the search engines:
```

File: pages/common/onionsearch.md [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+`onionsearch "Epstein"`
```
Comment (kbdharun) [inline pages/common/onionsearch.md:10]:
```suggestion
`onionsearch "{{string}}"`
```

File: pages/common/onionsearch.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+- Request search results from specific search engines:
+
+`onionsearch "{{Lolita Island}}" --engines {{tor66 deeplink phobos ...}}`
```
Comment (kbdharun) [inline pages/common/onionsearch.md:14]:
```suggestion
`onionsearch "{{string}}" --engines {{tor66 deeplink phobos ...}}`
```

File: pages/common/onionsearch.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+- Exclude certain serarch engines when searching:
+
+`onionsearch "{{Hunter Biden}}" --exclude {{candle ahmia ...}}`
+
+- Limit the number of pages to load per engine:
+
+`onionsearch "{{stuxnet}}" --engines {{tor66 deeplink phobos ...}} --limit {{3}}`
+
+- List of supported search engines:
```
Comment (kbdharun) [inline pages/common/onionsearch.md:24]:
```suggestion
- List all supported search engines:
```

File: pages/common/onionsearch.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+`onionsearch "Epstein"`
+
+- Request search results from specific search engines:
+
+`onionsearch "{{Lolita Island}}" --engines {{tor66 deeplink phobos ...}}`
+
+- Exclude certain serarch engines when searching:
+
+`onionsearch "{{Hunter Biden}}" --exclude {{candle ahmia ...}}`
```
Comment (kbdharun) [inline pages/common/onionsearch.md:18]:
```suggestion
`onionsearch "{{string}}" --exclude {{candle ahmia ...}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution, I have some suggestions for this page.

File: pages/common/onionsearch.md [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+
+- Request results from all the search engines:
+
+`onionsearch "{{string}}"`
+
+- Request search results from specific search engines:
+
+`onionsearch "{{string}}" --engines {{tor66 deeplink phobos ...}}`
+
+- Exclude certain serarch engines when searching:
```
Comment (spageektti) [inline pages/common/onionsearch.md:15]:
```suggestion
- Exclude certain search engines when searching:
```

File: pages/common/onionsearch.md [hunk 9]
```diff
@@ -0,0 +1,25 @@
+# onionsearch
+
+> Scrape URLs on different `.onion` search engines.
+> Note: `onionsearch` requires tor proxy running on `localhost:9050` and a tor enabled browser is needed to visit the `.onion` websites.
```
Comment (spageektti) [inline pages/common/onionsearch.md:4]:
```suggestion
> Note: `onionsearch` requires a Tor proxy running on `localhost:9050`; a Tor enabled browser is needed to visit the `.onion` websites.
```
Tor is capitalized like this on its website and Wikipedia.

**Review by spageektti — APPROVED** [review]

**Pull request merged**